src/components/EventManagement/AllEvents.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from "react";
+import { NavLink, useNavigate } from "react-router-dom";
+import { getEventdata, eventDelete } from "../../api";
+import _ from "lodash";
```

Comment:
never import complete lodash to use in a component as it increase the build size. Please import the utility from lodash which is needed,
```
import map from "lodash/map";
```

Reply:
done